certification/internal/shell/engine.go

```diff
@@ -25,6 +26,7 @@ type CheckEngine struct {
 func (e *CheckEngine) ExecuteChecks(logger *logrus.Logger) {
 	logger.Info("target image: ", e.Image)
 	for _, check := range e.Checks {
+		checkStartTime := time.Now()
```

Comment:
Do we want to include the image download time in the elapsed time? I think this will be misleading from the end user perspective, in particular if we are implementing the caching mechanism: the first check will take longer, and the subsequent ones are going to be fast because the image is on the local machine already. I believe we should separate the image download from the validation.

Reply:
Good point. I'll work around that.